src/functionalTest/groovy/com/autonomousapps/android/AbstractAndroidSpec.groovy

```diff
@@ -25,21 +25,23 @@ abstract class AbstractAndroidSpec extends AbstractFunctionalSpec {
   protected static final AGP_8_11 = AgpVersion.version('8.11.1')
   protected static final AGP_8_12 = AgpVersion.version('8.12.2')
   protected static final AGP_8_13 = AgpVersion.version('8.13.0')
+  protected static final AGP_9_0 = AgpVersion.version('9.0.0-alpha04')
 
-  protected static final AGP_LATEST = AGP_8_13
+  protected static final AGP_LATEST = AGP_9_0
```

Comment:
We can track latest AGP like

https://github.com/vanniktech/gradle-maven-publish-plugin/blob/d0e50fec7a07ef91fde62ae5cb0e4b4b4769d797/plugin/build.gradle.kts#L65-L69
https://github.com/vanniktech/gradle-maven-publish-plugin/blob/d0e50fec7a07ef91fde62ae5cb0e4b4b4769d797/gradle/alpha.versions.toml#L6
https://github.com/vanniktech/gradle-maven-publish-plugin/blob/d0e50fec7a07ef91fde62ae5cb0e4b4b4769d797/renovate.json#L19-L22